### ISSUE TITLE
Modified dynamic labral shear test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10995]
**Preferred term label:**
Modified dynamic labral shear test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sensation of a pain with or without clicking or clunking sensation with the following test. Flex patients elbow to 90° and abduct the arm 120° while maximally externally rotating the arm, then examiner lowers the arm to 60° of abduction. A positive test occurs with pain and/or a painful click along the posterior joint line between 120° and 90° of abduction

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Labrum tear, shoulder

**Your nano-attribution (ORCID)**
https://cdn-links.lww.com/permalink/jaaos/a/jaaos_2018_09_17_hippensteel_jaaos-d-17-00637_sdc12.mp4

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001226
- label: Positive modified dynamic labral shear test
- parents: [Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: PMID:19509414
- def: Sensation of a pain with or without clicking or clunking sensation with the following test. Flex patients elbow to 90 degrees and abduct the arm 120 degrees while maximally externally rotating the arm, then examiner lowers the arm to 60 degrees of abduction. A positive test occurs with pain and/or a painful click along the posterior joint line between 120 degrees and 90 degrees of abduction.
- comment: This test is frequently positive with labrum tear of the shoulder.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10995
- synonyms:
